bucket/catalyst.json

```diff
@@ -0,0 +1,9 @@
+{ 
+    "version": "3.2.2",
+    "homepage": "https://getcatatalyst.eu.org",
+    "license": "MIT",
+    "notes": "As of the past few releases, 32-bit support has been unavailable due to the lack of Electron support for such architecture.",
+    "url": "https://github.com/JaydenDev/Catalyst/releases/download/v3.2.2/catalyst-3.2.2.Setup.exe",
+    "hash": "sha256:9d950fb4a4ad449fd8df4e8f12fc1b1d31e4d9fcaf93d3bb2bb1767b86636f26",
+    "bin": "catalyst-3.2.2.Setup.exe"
```

Comment:
That is a setup program, so this needs to be unpacked. See other manifests for examples.

Reply:
@rasa This executable was build with [Electron Forge](https://www.electronforge.io/) complicating the process, additional information is needed.